phpBB/phpbb/forum/controller/viewforum.php

```diff
@@ -134,32 +161,22 @@ protected function render_forum($forum_id, $parameters)
 				redirect($forum_data['forum_link'], false, true);
 			}
 
-			$forum_parents = $this->forum_retriever->get_forum_parents($forum_data);
-
-			// This is mostly done for BC reasons...
-			$forum_data['forum_parents'] = serialize($forum_parents);
-
-			$forum_parents = $this->visibility_helper->filter_forum_parents($forum_parents);
-
-			if ($forum_data['left_id'] != $forum_data['right_id'] - 1)
+			$has_read_access = $this->auth->acl_gets('f_read', 'f_list_topics', $forum_id);
+			if (!($forum_data['forum_type'] == FORUM_POST ||
+				(($forum_data['forum_flags'] & FORUM_FLAG_ACTIVE_TOPICS) && $forum_data['forum_type'] == FORUM_CAT)) ||
+				!$has_read_access)
```

Comment:
Same for multiple line `if`statements.
Perhaps there should be added something about it in the guidelines (if I am not missing it already)?

```php
if (
	!($forum_data['forum_type'] == FORUM_POST ||
	(($forum_data['forum_flags'] & FORUM_FLAG_ACTIVE_TOPICS) && $forum_data['forum_type'] == FORUM_CAT)) ||
	!$has_read_access
)
{
```